integration/kubernetes/multiMaster-embeddedJournal/master/alluxio-master-service.yaml.template

```diff
@@ -41,17 +29,30 @@ spec:
     role: alluxio-master
     app: alluxio
     release: alluxio
-    statefulset.kubernetes.io/pod-name: alluxio-master-0
+    statefulset.kubernetes.io/pod-name: alluxio-master-2
 ---
+# Source: alluxio/templates/master/service.yaml
+#
+# The Alluxio Open Foundation licenses this work under the Apache License, version 2.0
+# (the "License"). You may not use this work except in compliance with the License, which is
+# available at www.apache.org/licenses/LICENSE-2.0
+#
+# This software is distributed on an "AS IS" basis, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+# either express or implied, as more fully set forth in the License.
+#
+# See the NOTICE file distributed with this work for information regarding copyright ownership.
+#
+
+
```

Comment:
Seems Helm 3 has its own understanding of where the comments should appear in a range loop. This is really hilarious. @madanadit